extensions/iceberg/src/main/java/io/deephaven/iceberg/util/IcebergUpdateMode.java

```diff
@@ -0,0 +1,57 @@
+//
+// Copyright (c) 2016-2024 Deephaven Data Labs and Patent Pending
+//
+package io.deephaven.iceberg.util;
+
+import io.deephaven.annotations.BuildableStyle;
+import org.immutables.value.Value;
+
+@Value.Immutable
+@BuildableStyle
+public abstract class IcebergUpdateMode {
+    private static final long REFRESH_INTERVAL_MS = 60_000L;
+
+    public static final IcebergUpdateMode STATIC = builder().updateType(IcebergUpdateType.STATIC).build();
```

Comment:
I would probably make these private and access them via public static methods, just to keep a common/conventional style with `autoRefreshing(long)`.